packages/ckeditor5-paragraph/src/paragraph.js

```diff
@@ -67,6 +67,7 @@ export default class Paragraph extends Plugin {
 
 				return modelWriter.createElement( 'paragraph' );
 			},
+			view: /\S/, // TODO: i/7336 - better API.
```

Comment:
```suggestion
			view: /\S/, // TODO: i/7336 - better API.
```

Ugh, left-over. I'll check the proper matcher option for this (catch-all).

Reply:
We've used `/[\s\S]+/` for attributes matching, so I'd used that for now.

Reply:
And this is also "wrong" - there is a set of characters that can be used but `\n` is not one of them. Using `.+` is enough.